internal/database/assigned_owners.go

```diff
@@ -75,6 +73,35 @@ func (s assignedOwnersStore) ListAssignedOwnersForRepo(ctx context.Context, repo
 	return scanAssignedOwners(s.Query(ctx, q))
 }
 
+const deleteAssignedOwnerFmtstr = `
+	WITH repo_path AS (
+		SELECT id
+		FROM repo_paths
+		WHERE absolute_path = %s AND repo_id = %s
+	)
+	DELETE FROM assigned_owners
+	WHERE owner_user_id = %s AND file_path_id = (
+		SELECT p.id
+		FROM repo_path AS p
+	)
```

Comment:
FYI this can be done with somewhat shorter `USING` syntax (which as I understand is like old-style-join) for delete.

```suggestion
	DELETE FROM assigned_owners AS o
	USING repo_paths AS p
	WHERE o.file_path_id = p.id
		AND p.absolute_path = %s
		AND p.repo_id = %s
		AND o.owner_user_id = %s
```

Reply:
TIL!